app/views/simulator/edit.html.erb

```diff
@@ -297,4 +297,46 @@
       <script src="/js/Quin_McCluskey.js"></script>
     <% end %>
     <script src="/js/canvas2svg.js"></script>
-    <div>
+    
+    <!---issue reporting-system----->
+    <div class="mysidebar">
```

Comment:
`mysidebar` is misleading, we already have a `sidebar`. Please use names like `report-sidebar` or `feedback-sidebar`

Reply:
will change this soon